docs/asset-manifest.json

```diff
@@ -1,10 +1,11 @@
 {
```

Comment:
Let's not change the contents inside the `docs` as we keep that for deployment purposes.